lib/util/fsUtil.js

```diff
@@ -1,10 +1,10 @@
 const path = require('path');
 
+const sourceFileExtensions = ['.html', '.md', '.mbd', '.mbdf'];
```

Comment:
`*FileExts` seems like an abbrievation of `*FileExtensions`.
Also, our `utils.getExtName` returns without `.` while `path.extname` returns with.

Let's rename `utils.getExtName` to `utils.getExt`.
And then rename this to `sourceFileExtNames`.

`isSourceFile` and `isMarkdownFileExt` are fine.